test/celeritas/ext/Vecgeom.test.cc

```diff
@@ -30,10 +31,17 @@ using namespace celeritas_test;
 #    define TEST_IF_CELERITAS_CUDA(name) DISABLED_##name
 #endif
 
+// Always compile and sometimes disable tests that require Geant4
+#if CELERITAS_USE_GEANT4
+#    define TEST_IF_CELERITAS_GEANT(name) name
+#else
+#    define TEST_IF_CELERITAS_GEANT(name) DISABLED_##name
+#endif
+
 //---------------------------------------------------------------------------//
 // TESTS
 //---------------------------------------------------------------------------//
-class VecgeomTest : public celeritas_test::GlobalTestBase
+class VecgeomTestBase : virtual public celeritas_test::GlobalTestBase
```

Comment:
Why not a separate file for VecgeomTestBase, like most other `TestBase`s?

Reply:
It's only used in this one file: if we used it in multiple test files we'd need it separately.